bl/drawable/src/drawable.cpp

```diff
@@ -9,21 +9,23 @@ void Drawable::addChild(const std::shared_ptr<Drawable> &child, Drawable::Positi
     children[child] = p;
 }
 
-Drawable::Drawable(const Drawable &d) {
-    color = d.color;
-    shape = d.shape;
-    children = d.children;
-}
+void Drawable::removeChild(const std::shared_ptr<Drawable> &child) { children.erase(child); }
```

Comment:
shouldn't be in this PR it is not fixing any clang-tidy issue

Reply:
fixed